### PR TITLE
Blog App : Views

### PR DESCRIPTION
**Pull request by Piercel2022**
This pull request is open for the view branch to display the View after the Model and the  Controller was made.
The design is implemented from the sneak peek wireframes. The data that is displaying here in the view are the database records that was entered before when I implemented the database. So now the view show these records.

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
-<p> <%= @post.id %> </p>
-<p> <%= @post.author_id %> </p>
-<p> <%= @post.title %> </p>
-<p> <%= @post.text %> </p>
\ No newline at end of file
+<div>
+  <ul>
+    <% @posts.each do |post| %>
+      <li>
+        <h3 class="mb-2 text-sm font-bold"><%= post.title %> #<%= post.id %></h3>
```
Comment (FrederickMih) [inline app/views/posts/index.html.erb:7]: - [ ] Let's make sure we are taken to the post-show page by clicking on each post.

File: app/views/posts/show.html.erb [hunk 2]
```diff
@@ -1 +1,14 @@
-<h1> Placeholder for the post</h1>
\ No newline at end of file
+<<h1>Show post</h1>
```
Comment (FrederickMih) [inline app/views/posts/show.html.erb:1]: - [ ] I didn't reach this page. I could not get here by clicking or manually routing on the browser using the URL. I got the console error below.


Screenshot from 2022-11-17 04-17-29

- Let's kindly fix :wrench: the error.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @Piercel2022, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



Screenshot from 2022-10-20 09-38-51

## Highlights :green_circle: 

- Descriptive README.md file :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- PR has a good title and summary :heavy_check_mark: 
- Correct GitHub flow :+1: 



## Required Changes ♻️
N/A
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @Piercel2022, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**